mti_nma/steps/avgshape/avgshape.py

```diff
@@ -26,29 +25,32 @@ def __init__(
         self,
         direct_upstream_tasks: Optional[List["Step"]] = [Shparam],
         filepath_columns=["AvgShapeFilePath", "AvgShapeFilePathStl"]
+
     ):
         super().__init__(
             direct_upstream_tasks=direct_upstream_tasks,
             filepath_columns=filepath_columns
         )
 
     @log_run_params
-    def run(self, mesh_density=5, **kwargs):
+    def run(self, mesh_density=5, sh_df=None, **kwargs):
         """
         This step uses the amplitudes of the spherical harmonic components
         of the nuclear shapes in the dataset to construct an average nuclear mesh.
+
+        Parameters
+        ----------
+        mesh_density: int (1-10)
+            Mesh density parameter used in Blender 
         """
 
```

Comment:
If you want to be able to run this step in the CLI I would recommend adding something like [this default setter](https://github.com/AllenCellModeling/example_step_workflow/blob/master/example_step_workflow/steps/mapped_invert/mapped_invert.py#L80)

Reply:
Cool! Added.